"APIv2 Docs:" [opentrons/new_atomic_commands.rst] (#4098)
Opentrons/opentrons: api/docs/source/v2/new_atomic_commands.rst

This loads a `Corning 96 Well Plate <https://labware.opentrons.com/corning_96_wellplate_360ul_flat>`_ in slot 2 and a `Opentrons 300ul Tiprack <https://labware.opentrons.com/opentrons_96_tiprack_300ul>`_ in slot 3, and uses a P300 Single pipette.

> Iterating With Tips

Reviewer: I don't think I agree with this, I think "through" is better; but also "iterating" might not be the best word here.

Aspirate
========

> When calling :py:meth:`.InstrumentContext.aspirate` on a pipette, we can specify how many microliters, and at which location, to draw liquid from, this aspiratation will pull liquid up into the pipette's tip:

Reviewer: ```suggestion
When calling :py:meth:`.InstrumentContext.aspirate` on a pipette, we can specify how many microliters, and at which location, to draw liquid from, this aspiration will pull liquid up into the pipette's tip:
```
?

.. code-block:: python

    pipette.aspirate(50, plate['A1'])  # aspirate 50uL from plate:A1

Now our pipette's tip is holding 50uL.

> We can also simply specify how many microliters to aspirate, without specifiying a location. The pipette in this circumstance will aspirate from its current location (which we previously set as ``plate['A1'])``.

Reviewer: ```suggestion
We can also simply specify how many microliters to aspirate, without specifying a location. The pipette in this circumstance will aspirate from its current location (which we previously set as ``plate['A1']``).
```

.. note::

    By default, the OT-2 will move to 1mm above the bottom of the target before aspirating.
    You can specify a well position function like :py:meth:`.Well.bottom` (see

>     :ref:`v2-location-within-wells`) every time you call ``aspirate``, or -  change

Reviewer: ```suggestion
    :ref:`v2-location-within-wells`) every time you call ``aspirate``, or change
```

I think this is a leftover dash?

Blow Out
========

In order to ensure that any remaining droplets are expelled, a blow out will push air through the pipette's tip.

> When calling :py:meth:`.InstrumentContext.blow_out`, we have the option to specify a location expell remaining liquid. If no location is specified, the pipette will blow out from its current position.

Reviewer: ```suggestion
When calling :py:meth:`.InstrumentContext.blow_out`, we have the option to specify where to expel remaining liquid. If no location is specified, the pipette will blow out from its current position.
```

Mix
===

Mixing performs ``aspirate()`` and ``dispense()`` on a single location. However, instead of having to write those commands out every time, the Opentrons API allows you to simply say :py:meth:`.InstrumentContext.mix`.

> Three arguements are possible utilizing the mix command: ``mix(repetitions, volume, location)``

Reviewer: ```suggestion
Three arguments are possible utilizing the mix command: ``mix(repetitions, volume, location)``
```

Air Gap
=======

> Some liquids need an extra amount of air in the pipette's tip to prevent it from sliding out. By calling :py:meth:`.InstrumentContext.air_gap` and specifying a microliter amount, aspiratation of that much air will be released into pipette's tip.

Reviewer: ```suggestion
Some liquids need an extra amount of air in the pipette's tip to prevent it from sliding out. By calling :py:meth:`.InstrumentContext.air_gap` and specifying a microliter amount, aspiration of that much air will be released into pipette's tip.
```

    pipette.move_to(plate['A1'].bottom(2)) # move to 2mm above the bottom of well A1
    pipette.move_to(plate['A1'].top(-2))   # move to 2mm below the top of well A1

The above commands will cause the robot's head to move upwards, then above the target location, then downwards until the target location is reached.

To move the robot in a straight line to the target location, you can set the movement strategy to ``'direct'``.

.. code-block:: python

    pipette.move_to(plate['A1'], force_direct=True)

.. warning::

>     Note: Moving without an arc will run the risk of colliding with things on your deck. 

Reviewer: ```suggestion
    Moving without an arc will run the risk of colliding with things on your deck. 
```

The `.. warning::` line above sets the following text out in a background-colored box with a big warning title so I don't think we need the `Note: ` here. See https://docs.opentrons.com/v2/new_atomic_commands.html, this is what it looks like:

<img width="698" alt="Screen Shot 2019-09-24 at 10 29 30 AM" src="https://user-images.githubusercontent.com/3091648/65520837-41270080-deb6-11e9-94c0-cb84137964e6.png">

Reviewer: @wheresaddie I'm a little confused about this `Note: ` still. Keep in mind that the image pasted there does _not_ include the change. With the change, you'd both have the big red box that starts with `Warning:` and then the text itself starting with `Note: `. Is this intentional?

====================

The method :py:meth:`.ProtocolContext.pause` will pause protocol execution at a specific step.
You can resume by pressing 'resume' in your OT App. 

> You can optionally specify a message thatwill be displayed in the Opentrons app when protocol execution pauses:

Reviewer: ```suggestion
You can optionally specify a message that will be displayed in the Opentrons app when protocol execution pauses:
```
